tests/system/small/test_pandas.py

```diff
@@ -45,6 +45,56 @@ def test_concat_series(scalars_dfs):
     pd.testing.assert_series_equal(bf_result, pd_result)
 
 
+@pytest.mark.parametrize(
+    ("args"),
+    [
+        [["prefix1", "prefix2"], "_", None, ["bool_col", "int64_col"], False],
+        ["prefix", ["_", ","], False, ["int64_too", "string_col"], False],
+        [None, ".", True, ["time_col", "float64_col"], True],
+    ],
+)
+def test_get_dummies_dataframe(scalars_dfs, args):
+    scalars_df, scalars_pandas_df = scalars_dfs
+    prefix, prefix_sep, dummy_na, columns, drop_first = args
+
+    bf_result = bpd.get_dummies(
+        scalars_df,
+        prefix=prefix,
+        prefix_sep=prefix_sep,
+        dummy_na=dummy_na,
+        columns=columns,
+        drop_first=drop_first,
+    )
+    pd_result = pd.get_dummies(
+        scalars_pandas_df,
+        prefix=prefix,
```

Comment:
[nit] would be more assured (and concise) to define once and use twice:

```
args = {
        'prefix' : prefix,
        'prefix_sep' : prefix_sep,
        'dummy_na' : dummy_na,
        'columns' : columns,
        'drop_first' : drop_first,
}

bf_result = bpd.get_dummies(scalars_df, **args)
pd_result = pd.get_dummies(scalars_pandas_df, **args)
```

Reply:
Good suggestion, this makes the body of the tests simpler and although the decorator is a little bit verbose it's also clear.